roles/ncn-common-setup/tasks/metal.yml

```diff
@@ -17,3 +17,14 @@
     remote_src: yes
     src: /srv/cray/resources/metal/sshd_config
     dest: /etc/ssh/sshd_config
+
+- name: Setup dracut to use fstab.metal
```

Comment:
Nit: let's change this to say why

```yaml
- name: Update Dracut to include dynamically created partitions from deployments
```

Just my $0.02 but that would explain to another maintainer why we care about adding fstab.metal.